src/ott/initializers/linear/initializers.py

```diff
@@ -132,32 +151,35 @@ def init_dual_b(
 class GaussianInitializer(DefaultInitializer):
   """Gaussian initializer :cite:`thornton2022rethinking:22`.
 
-  Compute Gaussian approximations of each point cloud, then compute closed from
-  Kantorovich potential between Gaussian approximations using Brenier's theorem
-  (adapt convex/Brenier potential to Kantorovich). Use this Gaussian potential
-  to initialize Sinkhorn potentials/scalings.
+  Compute Gaussian approximations of each :class:`~ott.geometry.pointcloud.PointCloud`,
+  then compute closed from Kantorovich potential between Gaussian approximations using
+  Brenier's theorem (adapt convex/Brenier potential to Kantorovich).
+  Use this Gaussian potential to initialize Sinkhorn potentials/scalings.
   """
 
   def init_dual_a(
       self,
-      ot_prob: 'linear_problem.LinearProblem',
+      ot_prob: linear_problem.LinearProblem,
       lse_mode: bool,
+      rng: Optional[jax.random.PRNGKeyArray] = jax.random.PRNGKey(0)
   ) -> jnp.ndarray:
     """Gaussian initialization function.
 
     Args:
       ot_prob: OT problem between discrete distributions of size n and m.
       lse_mode: Return potential if true, scaling if false.
+      rng: Random number generator, not needed for this initializer.
 
     Returns:
       potential/scaling, array of size n.
     """
+    del rng
     # import Gaussian here due to circular imports
     from ott.tools.gaussian_mixture import gaussian
 
     assert isinstance(
         ot_prob.geom, pointcloud.PointCloud
-    ), "Gaussian initializer valid only for point clouds."
+    ), "Gaussian initializer valid only for :class:`~ott.geometry.pointcloud.PointCloud` geoms."
```

Comment:
Please revert this, the :class: role is meant only for docs (this will produce ugly messages); I assume this is an artifact of search-and-replace (also in other places please).